week-1/Homework/mandatory/4-tax.js

```diff
@@ -5,7 +5,9 @@
   Sales tax is 20% of the price of the product
 */
 
-function calculateSalesTax() {}
+function calculateSalesTax(price) {
+  return price + (price*0.2)
+}
```

Comment:
You could have done differently by storing calculation in a variable. e.g let taxAmount = price * 0.2; and a the price plus the amount tax as let total = price + taxAmount; then return total.